public/content/guides/how-to-create-an-ethereum-account/index.md

```diff
@@ -30,43 +30,43 @@ Once you have decided on a specific wallet, visit their official website or app
+Some apps will request you to save a secret 'recovery phrase' (sometimes called a "seed phrase" or a "mnemonic"). Keeping this phrase safe is extremely important! This is used to generate your Ethereum account and can be used to submit transactions.
 
-Once you have saved your seed phrase you should see your wallet dashboard with your balance. Check out our guide: [how to use a wallet.](/guides/how-to-use-a-wallet)
+**Any person who knows the phrase can take control of all funds.** Never share this with anyone. This phrase should contain 12 to 24 randomly generated words (the order of the words matters).
 
- <br />
- 
+<div>
 <InfoBanner shouldSpaceBetween emoji=":eyes:">
-  <div>Want to learn more?</div>
-  <ButtonLink to="/guides/">
-    See our other guides
+  <div><b>Wallet installed?</b><br/>Learn how to use it.</div>
+  <ButtonLink to="/guides/how-to-use-a-wallet">
+    How to use a wallet
   </ButtonLink>
 </InfoBanner>
+</div>
+
+Interested in other guides? Check out our: [Step by step guides](/guides/)
 
 ## Frequently asked questions
 
 ### Are my wallet and my Ethereum account the same?
 
-No. The wallet is a management tool that helps you to manage accounts. A single wallet might give access to several accounts, and a single account can be accessed by multiple wallets. The seed phrase is used to create accounts that are then controlled by the wallet.
-
-You can think of the accounts as leaves on a tree that all 'grow' from a single seed phrase. Each unique seed will grow an entirely different tree of accounts.
+No. The wallet is a management tool that helps you to manage accounts. A single wallet might access several accounts, and a single account can be accessed by multiple wallets. The recovery phrase is used to create accounts and gives permission to a wallet app to manage assets.
 
 ### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
 
-No, you cannot. Bitcoin and ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping models and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
+No, you cannot. Bitcoin and ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
```

Comment:
An aside, would suggest we just follow the capitalization used on the bitcoin.com website for WBTC: "Wrapped Bitcoin"

```suggestion
No, you cannot. Bitcoin and ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped Bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
```